core/kazoo_schemas/src/kz_json_schema_extensions.erl

```diff
@@ -115,3 +116,43 @@ validate_attachment_oauth_doc_id(Value, State) ->
             lager:debug("~s", [ErrorMsg]),
             jesse_error:handle_data_invalid('external_error', ErrorMsg, State)
     end.
+
+maybe_check_params_stability_level(Value, Schema, State) ->
+    case {kz_json:get_value(<<"properties">>, Schema),
+          kz_json:get_value(<<"id">>, Schema),
+          kz_json:is_json_object(Value)}of
+        {'undefined', _, _} ->
+            State; %% Skipping schema with out `properties' property
+        {Properties, Id, true} when Id /= <<"system_config.crossbar">> ->
+            check_params_stability_level(Properties, Value, State);
+        _ ->
+            State
+    end.
+
+check_params_stability_level(Properties, Value, State) ->
+    SystemSL = kapps_config:get_binary(<<"crossbar">>, <<"stability_level">>, <<"stable">>),
+    SystemSLInt = stability_level_to_int(SystemSL),
+    Fun = fun(Key, _Val, Acc) ->
+                  SearchKey = [Key, <<"stability_level">>],
+                  PropSL = kz_json:get_value(SearchKey, Properties, <<"stable">>),
+                  case stability_level_to_int(PropSL) < SystemSLInt of
```

Comment:
Don't forget to change this line, since github thinks it changes. Look at my previous comment about it.